src/main/webapp/resources/js/pages/projects/linelist/components/Table/renderers/SampleNameRenderer.jsx

```diff
@@ -1,5 +1,4 @@
 import React from "react";
```

Comment:
File comment

Reply:
Added in 8f893d85e7b08437dc011c37a55c916eef471d08